communitybridge/2020/q3-q4/project_ideas.md

```diff
@@ -101,4 +101,26 @@ Project maintainers and mentors, please submit the ideas below (under the Propos
 -	Description: Improve Job API to support specific types of job
 -	Recommended Skills: Volcano, Job Management, Golang
 -	Mentor(s): Zhonghu Xu(@hzxuzhonghu), Klaus Ma(@k82cn) 
--	Upstream Issue (URL): https://github.com/volcano-sh/volcano/issues/1034
\ No newline at end of file
+-	Upstream Issue (URL): https://github.com/volcano-sh/volcano/issues/1034
+
+#### Thanos
+##### Receive Controller (tentative)
```

Comment:
What exactly is the project here and how does it differ from 2?

Reply:
We are going to merge two projects and at the end will have one for the receiver component. I just created this draft to initialize to process. We're gonna finalize it today.

Reply:
nice that sounds good

Reply:
I spoke with @kakkoyun, this was a misunderstanding I think. There was no project for TRC, but rather one for Thanos Receive.